src/Android/Xamarin.Android/Samples/GraphicsOverlay/AddGraphicsWithSymbols/AddGraphicsWithSymbols.cs

```diff
@@ -0,0 +1,171 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System.Drawing;
+
+namespace ArcGISRuntimeXamarin.Samples.AddGraphicsWithSymbols
+{
+    [Activity]
+    public class AddGraphicsWithSymbols : Activity
+    {
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+
+        // Create the graphics overlay
+        private readonly GraphicsOverlay _overlay = new GraphicsOverlay();
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Add graphics with symbols";
+
+            // Create the UI, setup the control references and execute initialization
+            CreateLayout();
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create the map
+            Map myMap = new Map(BasemapType.Oceans, 56.075844, -2.681572, 13);
+
+            // Add the map to the map view
+            _myMapView.Map = myMap;
+
+            // Add the graphics overlay to the map view
+            _myMapView.GraphicsOverlays.Add(_overlay);
+
+            // Call functions to create the graphics
+            CreatePoints();
+            CreatePolygon();
+            CreatePolyline();
+            CreateText();
+        }
```

Comment:
I would recommend adding the following code after the last set of graphics have been added to the GraphicsOverlays to get the extent of all the graphics and zooming the map to that extent.  This way all of the graphics appear when the sample loads under all device form factors.

            // Get all of the graphics contained in the graphics overlay
            GraphicCollection myGraphicCollection = _overlay.Graphics;

            // Create a new envelope builder using the same spatial reference as the graphics
            EnvelopeBuilder myEnvelopeBuilder = new EnvelopeBuilder(SpatialReferences.Wgs84);

            // Loop through each graphic in the graphic collection
            foreach (Graphic oneGraphic in myGraphicCollection)
            {
                // Union the extent of each graphic in the envelope builder
                myEnvelopeBuilder.UnionOf(oneGraphic.Geometry.Extent);
            }

            // Expand the envelope builder by 30%
            myEnvelopeBuilder.Expand(1.3);

            // Adjust the viewable area of the map to encompass all of the graphics in the
            // graphics overlay plus an extra 30% margin for better viewing
            _myMapView.SetViewpointAsync(new Viewpoint(myEnvelopeBuilder.Extent));